tests/test_opentok.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from requests.models import Response
+
+from six import text_type, u, b, PY2, PY3
+from nose.tools import raises
+
+import requests
+import httpretty
+from sure import expect
```

Comment:
Looks like we need to add this as a dev dependency, the CI is complaining the module doesn't exist